pom.xml

```diff
@@ -119,7 +119,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.integration.common</groupId>
       <artifactId>event-publisher</artifactId>
-      <version>0.0.25</version>      
+      <version>0.0.27-SNAPSHOT</version>
```

Comment:
Not in a PR please

Reply:
Addressed. I shall take myself outside and pelt myself with rotten tomatoes later.